core/dpdk.cc

```diff
@@ -51,32 +51,6 @@
 namespace bess {
 namespace {
 
-int get_numa_count() {
-  FILE *fp;
-
-  int matched;
-  int cnt;
-
-  fp = fopen("/sys/devices/system/node/possible", "r");
-  if (!fp) {
-    goto fail;
-  }
-
-  matched = fscanf(fp, "0-%d", &cnt);
-  if (matched == 1) {
-    return cnt + 1;
-  }
-
-fail:
-  if (fp) {
-    fclose(fp);
-  }
-
-  LOG(INFO) << "/sys/devices/system/node/possible not available. "
-            << "Assuming a single-node system...";
-  return 1;
-}
-
```

Comment:
This whole commit could be folded back into the original one that adds `memory.cc`...

Reply:
:+1: